staging/src/kubevirt.io/client-go/kubecli/migration.go

```diff
@@ -148,12 +102,12 @@ func (v *migration) PatchStatus(name string, pt types.PatchType, data []byte) (r
 	return
 }
 
-func (v *migration) UpdateStatus(vmi *v1.VirtualMachineInstanceMigration) (result *v1.VirtualMachineInstanceMigration, err error) {
+func (o *migration) UpdateStatus(vmi *v1.VirtualMachineInstanceMigration) (result *v1.VirtualMachineInstanceMigration, err error) {
 	result = &v1.VirtualMachineInstanceMigration{}
-	err = v.restClient.Put().
+	err = o.restClient.Put().
 		Name(vmi.ObjectMeta.Name).
-		Namespace(v.namespace).
-		Resource(v.resource).
+		Namespace(o.namespace).
+		Resource(o.resource).
 		SubResource("status").
```

Comment:
I think you forgot to use ` o.VirtualMachineInstanceMigrationInterface.UpdateStatus` here

Reply:
Nice catch! You're right! Thanks for catching this! Done